examples/tutorials/core_api_pytorch_mnist/model_def_metrics.py

```diff
@@ -63,21 +73,29 @@ def train(args, model, device, train_loader, optimizer, epoch_idx, core_context)
                     len(train_loader.dataset),
                     100.0 * batch_idx / len(train_loader),
                     loss.item(),
+                    # Docs snippet end before: increment batch idx
                 )
             )
-
-            # NEW: Report epoch-based training metrics to Determined master via core_context.
-            # Index by (batch_idx + 1) * (epoch-1) * len(train_loader) to continuously plot loss on one graph for consecutive epochs.
+            # Docs snippet start after: report training metrics
+            # NEW: Report epoch-based training metrics to Determined
+            # master via core_context.
+            # Index by (batch_idx + 1) * (epoch-1) * len(train_loader)
+            # to continuously plot loss on one graph for consecutive
+            # epochs.
             core_context.train.report_training_metrics(
                 steps_completed=(batch_idx + 1) + epoch_idx * len(train_loader),
                 metrics={"train_loss": loss.item(), "epoch": epoch_idx},
+                # Docs snippet end before: report training metrics
```

Comment:
The closing paren on the next line closes the `report_training_metrics(` call, so you should include it in your literal include.

Reply:
moved docs snippet comment so that it is after the closing paren